redisinsight/ui/src/utils/transformers/convertDataRate.ts

```diff
@@ -0,0 +1,17 @@
+export function convertDataRate(kbps: number, options: { decimals?: number } = {}): { rate: number; unit: string } {
```

Comment:
can we use current formatBytes function?

Reply:
The initiate approached indeed used that but moved away from it due to getting some wrong numbers - https://github.com/RedisInsight/RedisInsight/pull/3827/commits/2140499be63f996464a50c4f917f0758438f5212

Let me explore it again.